packages/data-context/test/unit/data/ProjectConfigIpc.spec.ts

```diff
@@ -59,14 +58,16 @@ describe('ProjectConfigIpc', () => {
 
     context('typescript', () => {
       [...NODE_VERSIONS].forEach((nodeVersion) => {
+        const MOCK_NODE_PATH = `/Users/foo/.nvm/versions/node/v${nodeVersion}/bin/node`
+        const MOCK_NODE_VERSION = nodeVersion
+
         context(`node v${nodeVersion}`, () => {
-          context('ESM', () => {
-            it('passes the correct experimental flags if ESM is being used with typescript', async () => {
-              // @ts-expect-error
-              const projectPath = await scaffoldProject('config-cjs-and-esm/config-with-ts-module')
+          const PROJECTS = ['config-cjs-and-esm/config-with-ts-module', 'config-cjs-and-esm/config-with-module-resolution-bundler', 'config-cjs-and-esm/config-with-js-module', 'config-cjs-and-esm/config-with-cjs']
 
-              const MOCK_NODE_PATH = `/Users/foo/.nvm/versions/node/v${nodeVersion}/bin/node`
-              const MOCK_NODE_VERSION = nodeVersion
+          PROJECTS.forEach((project) => {
+            it(`${project}: tsx generic loader (esm/commonjs/typescript)`, async () => {
+              // @ts-expect-error
```

Comment:
Can you add a description to the ts-expect-error - usually we give some hint as to why this was needed in the comment

Reply:
yeah I can add something. it has to due with the nested directories in `config-cjs-and-esm` even though the scaffold is valid

Reply:
addressed in 4f0a50f66f85cee14a5c5da284243ebc2b84b658